### PR TITLE
Docs reorg

### PR DESCRIPTION
**Pull request by elena-kolevska**

### CODE REVIEW

File: docs/docs/bloom-filter.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+description: Bloom filters are a probabilistic data structure that checks for presence of an element in a set
+type: docs
+weight: 10
+stack: true
+aliases:
+    - /docs/stack/bloom/
+---
+
+
+# Bloom Filters
```
Comment (LiorKogan) [inline docs/docs/bloom-filter.md:13]: Bloom Filter  (singular)

File: docs/docs/bloom-filter.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+- Has the user already seen this ad? 
+- Has the user already bought this product?
+
+Use a Bloom filter for every user, storing all bought products. The recommendation engine suggests a new product and checks if the product is in the user's Bloom filter. 
+
+- If no, the ad is shown to the user and is added to the Bloom filter. 
+- If yes, the process restarts and repeats until it finds a product that is not present in the filter. 
+
+Using Redis Stack's bloom filters for this type of application provides these benefits:
```
Comment (LiorKogan) [inline docs/docs/bloom-filter.md:53]: Bloom (always with capital B)

File: docs/docs/count-min-sketch.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+linkTitle: Count-min sketch
+description: Count-min sketch is a probabilistic data structure that estimates the frequency of an element in a data stream.
+type: docs
+stack: true
+weight: 60
+---
+
+# Count-min sketch
+
+Count-Min Sketch is a probabilistic data structure in Redis Stack that estimates the frequency of events/elements in a stream of data.
```
Comment (LiorKogan) [inline docs/docs/count-min-sketch.md:12]: "that can be used to estimate"?

File: docs/docs/cuckoo-filter.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+linkTitle: Cuckoo filter
+description: Cuckoo filters are a probabilistic data structure that checks for presence of an element in a set
+type: docs
+weight: 20
+stack: true
+---
+
+# Cuckoo Filters
+
+Cuckoo filters, just like Bloom filters, are a probabilistic data structure in Redis Stack that enables you to check if an element is present in a set in a very fast and space efficient way, while also allowing for deletions and showing better performance than Bloom in some scenarios.
```
Comment (LiorKogan) [inline docs/docs/cuckoo-filter.md:12]: A cuckoo filter, just like a Bloom filter, is a ...

File: docs/docs/t-digest.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+---
+
+The t-digest is a sketch data structure in Redis Stack for estimating quantiles from a data stream or a large dataset using a compact sketch.
+
+It can answer questions like:
+- Which fraction of the values in the data stream are smaller than a given value?
+- How many values in the data stream are smaller than a given value?
+- What's the highest value that's smaller than *p* percent of the values in the data stream? (what is the p-percentile value)?
+
+### What is a quantile?
```
Comment (LiorKogan) [inline docs/docs/t-digest.md:17]: I really believe that we don't need this section.

File: docs/docs/t-digest.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
+---
+title: t-digest
+linkTitle: t-digest
+description: t-digest is a probabilistic data structure that allows you to estimate the percentile of a data stream.
+type: docs
+stack: true
+weight: 40
+---
+
+The t-digest is a sketch data structure in Redis Stack for estimating quantiles from a data stream or a large dataset using a compact sketch.
```
Comment (LiorKogan) [inline docs/docs/t-digest.md:10]: Replace "quantiles" with "percentiles"

**Review by LiorKogan — APPROVED** [review]

**Review by rafie — APPROVED** [review]

**Pull request merged**